pkgs/development/python-modules/mysql-connector/default.nix

```diff
@@ -43,8 +44,11 @@ buildPythonPackage rec {
   # Tests require a running MySQL instance
   doCheck = false;
 
+  nativeBuildInputs = lib.optionals stdenv.isDarwin [
+    gcc
```

Comment:
I'm not a big fan of this, I made  #177198 to solve this in a more correct way

Reply:
#177198 indeed looks better, thanks.

Reply:
I'm always a bit nervous with distro-specific patches. I hope the patch will eventually be merged upstream, otherwise we have to deal with conflicts in the future.